pom.xml

```diff
@@ -12,7 +12,7 @@
   <properties>
     <maven.compiler.target>11</maven.compiler.target>
     <maven.compiler.source>11</maven.compiler.source>
-    <version.drools-drl-ast>8.25.0.Beta</version.drools-drl-ast>
+    <version.drools-drl-ast>8.27.0.Beta</version.drools-drl-ast>
```

Comment:
Hi @tkobayas : is this 8.27 correct ?

Reply:
Yes, 8.28.0.Beta is not yet released at the moment :)